### PR TITLE
Katelyn/update workshop data

### LINKED ISSUE #305: Add new workshops
Update workshops page to have this year's workshops

### PR DESCRIPTION
**Pull request by katelynsyu**
Tis a new iteration of HOTH, so it's a new year of intern workshops. Updated the workshop page with the new workshops

Fixes #305

### CODE REVIEW

File: src/components/WorkshopPage/WorkshopPage.js [hunk 1]
```diff
@@ -42,70 +42,58 @@ const workshops = [
...
-				Let's talk about React, a JavaScript library used to build user interfaces! In this
-				workshop, you'll learn how to set up your own React App, as well as fundamental React
-				concepts such as components, props, and state. We'll even show you how to start building
-				your own version of Disney+. Let the magic begin!`,
-				youtube: 'https://youtu.be/4YZ0KZfCcb8',
-				readme: 'https://github.com/uclaacm/hoth9-workshops/tree/main/intro-to-reactjs',
-				slides: 'https://docs.google.com/presentation/d/1Q6-zbYOSTcvS5uN2GZHnxcpBYevpkj2HTGZImfVUEPc/edit?usp=sharing'
+				author: 'Brooke Jiang',
+				description: `In this workshop, we cover the basics of React with a simple demo
+				of creating a grocery list. We go over what React is and why we want to use it, make
```
Comment (nathanzzhang) [inline src/components/WorkshopPage/WorkshopPage.js:89]: "We will go over what"

File: src/components/WorkshopPage/WorkshopPage.js [hunk 2]
```diff
@@ -42,70 +42,58 @@ const workshops = [
...
+				our first component, and allow it to display different information using props and
+				array.map()! Furthermore, we learn the basics of State and useState, and how React magic
```
Comment (nathanzzhang) [inline src/components/WorkshopPage/WorkshopPage.js:91]: Furthermore, we will learn

File: src/components/WorkshopPage/WorkshopPage.js [hunk 3]
```diff
@@ -130,50 +118,53 @@ const workshops = [
...
 			{
 				title: 'Intro to Game Development',
-				author: 'Peter Sutarjo (ACM Studio)',
-				description: `In this workshop, we'll go over how to start thinking about your
-				prospective video game, such as whether you plan to create a visual novel or a
-				platformer. We'll give you some tips to succeed such as determining scope
-				and finding assets. At the end, we'll introduce you to some resources for
-				you to look at as you start diving deeper into developing your game.`,
-				youtube: 'https://youtu.be/D_mbel-Q5_0',
+				author: 'Austin Law',
```
Comment (nathanzzhang) [inline src/components/WorkshopPage/WorkshopPage.js:159]: do we want to attribute austin to acm studio? (idk where they are from)

**Review by nathanzzhang — CHANGES_REQUESTED** [review]
commented some minor blurb changes

File: src/components/WorkshopPage/WorkshopPage.js [hunk 4]
```diff
@@ -130,50 +118,53 @@ const workshops = [
...
-				first hybrid hackathon. We'll also give you some tips on how to survive and thrive during
-				this year's HOTH! Have fun, good luck, and we believe in you!`,
-				youtube: 'https://youtu.be/GskV5qMOpkE',
+				author: 'Anan Wang',
+				description: `This workshop videos explains the basics of hackathons, which are events where
+				people work together to create technology projects. You'll learn what to expect and how to make
+				the most out of your experience. By the end of the video, you'll have a better understanding of
+				what hackathons are all about and how to get started with attending one. Happy hacking, handy
+				hackers!`,
+				youtube: 'https://youtu.be/p8ssQybQsd8',
```
Comment (jamesmwu) [inline src/components/WorkshopPage/WorkshopPage.js:127]: "This workshop video explains"

File: src/components/WorkshopPage/WorkshopPage.js [hunk 5]
```diff
@@ -130,50 +118,53 @@ const workshops = [
...
-				workshop, we'll introduce git and GitHub (two very different things) with a
-				quick guide on how to set them up and how to use them. We'll simulate creating
-				a project with git and GitHub with all the intricacies of terminal, branching,
-				merging, and merge conflicts. Curious as to what those are? Wake up your galaxy brain,
-				watch this workshop, and 'git gud' at git!`,
-				youtube: 'https://youtu.be/YWEOSz7Lo8U',
-				readme: 'https://github.com/uclaacm/hoth9-workshops/tree/main/intro-to-git',
-				slides: 'https://docs.google.com/presentation/d/11Gb0WyIzhGyTfzxS0bg_IhUuvTgwYifyVpzmMK8aOmg/edit?usp=sharing'
+				author: 'Jenna Wang',
+				description: `Git is a distributed version control system; it brings a local copy of the
```
Comment (jamesmwu) [inline src/components/WorkshopPage/WorkshopPage.js:134]: Would consider adding a brief explanation on what that means for those who are totally new
"Git is a distributed version control system-- it helps programmers collaborate on projects! It does this by..."

**Review by jamesmwu — CHANGES_REQUESTED** [review]
Minor changes to descriptions, choose to change at your discretion. Everything else lgtm!

File: src/components/WorkshopPage/WorkshopPage.js (hunk 5 above)
Comment (katelynsyu) [reply src/components/WorkshopPage/WorkshopPage.js:134]: I would but I don't wanna change what Jenna wrote too much 🤷‍♀️

**Review by nathanzzhang — APPROVED** [review]
yeehaw lgtm

**Pull request merged**